e2e/run_tests.js

```diff
@@ -81,13 +81,16 @@ async function runTests() {
     await fse.remove('results');
     await fse.remove('screenshots');
 
+    const BROWSER = process.env.BROWSER ? process.env.BROWSER : "chrome";
     const testDirs = fse.readdirSync('cypress/integration/');
     let failedTests = 0;
 
     const mochawesomeReportDir = 'results/mochawesome-report';
 
     for (const dir of testDirs) {
         const {totalFailed} = await cypress.run({
+            browser: `${BROWSER}`,
+            headless: 'true',
```

Comment:
@srkgupta I'm setting `headless` to true for now. Although technically, it can be set to any non-empty string for it to be true (including "false" string), since the library code requires a key/value pair (but command line doesn't). The same goes for `headed` parameter. The headed/headless logic can be tackled later on by whoever's going to work on headed configs in cypress server. For now, I kept it simple.